html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email</label>
+        <div>
+          <input type="email" id="email" name="user_email" />
+        </div>
+      </div>
+      <div>
+        <label for="coding-select">Why is coding cool?</label>
+        <div>
+          <select id="why-is-coding-cool-select">
```

Comment:
This should be `id="coding-select"` to match your `for` attribute of the `label`